beast/plotting/plot_indiv_fit.py

```diff
@@ -332,7 +332,8 @@ def plot_beast_ifit(filters, waves, stats, pdf1d_hdu):
     starnum = args.starnum
 
     # base filename
-    filebase = args.filebase
+    #filebase = args.filebase
+    filebase = '/Users/clairemurray/beast/beast/examples/phat_small/beast_example_phat/beast_example_phat'
```

Comment:
This is a temporary override.  Should be deleted and the previous line uncommented.